com.github.brainstormtrooper.facteur.json

```diff
@@ -0,0 +1,52 @@
+{
+    "app-id" : "com.github.brainstormtrooper.facteur",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "44",
+    "branch" : "stable",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "com.github.brainstormtrooper.facteur",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--socket=system-bus",
+        "--socket=session-bus",
```

Comment:
these dbus permissions are NOT allowed. You need to determine which dbus names you need to access.